### DISCUSSION TITLE
Albert

### DISCUSSION THREAD

**Discussion by lighttigerXIV** [catppuccin/catppuccin#1951] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes
I made a theme for albert keyboard launcher using Catppuccin color scheme and i would like you to post it your repositorys.
Thanks :)

https://github.com/lighttigerXIV/albert

**Comment by elkrien** [comment]
Your repo readme should meet catppuccin standards first. Please use template for readme:

https://github.com/catppuccin/template

And style guide for other requirements (folders in repo etc.)

Then Poco81 will contact You with rest :)

**Comment by lighttigerXIV** [comment]
> 

I have updated my read.me. The only thing left is to change the links in the stars and contributors since it's not the final link

**Comment by jorgebef** [comment]
One small thing, in the Thanks to, I would include yourself and link to your Github.
I mean, you actually put the hard work in!

**Comment by nekowinston** [comment]
@lighttigerXIV we recently transferred your ulauncher repo to catppuccin, do you still also use Albert?
This would have to be updated to 0.2.0 before we can move it.

**Comment by lighttigerXIV** [comment]
> @lighttigerXIV we recently transferred your ulauncher repo to catppuccin, do you still also use Albert?
> This would have to be updated to 0.2.0 before we can move it.

I don't use albert anymore. I switched to ulauncher. If anyone does themes for albert I don't mind.
